cmd/syncthing/summaryservice.go

```diff
@@ -169,7 +169,8 @@ func (c *folderSummaryService) foldersToHandle() []string {
 	c.lastEventReqMut.Lock()
 	last := c.lastEventReq
 	c.lastEventReqMut.Unlock()
-	if time.Since(last) > pingEventInterval {
+	// TODO: Hard-coded value of 60, used to be pingEventInterval
```

Comment:
Add a constant `defaultEventTimeout` which is what we use when there is none given by the client, and also what we *expect* the GUI to use in order to keep triggering summary events?